shared/routes/Guide.html

```diff
@@ -193,7 +195,67 @@
 <script>
 	import GuideContents from '../components/GuideContents.html';
 
+	function throttle ( fn, ms ) {
+		let blocked = false;
+
+		function unblock () {
+			blocked = false;
+		};
+
+		return function () {
+			if ( blocked ) return;
+			blocked = true;
+
+			fn();
+			setTimeout( unblock, ms );
```

Comment:
I wonder whether this could lead to a memory leak if the timeouts aren't cleared.